lib/test_sites/cac.rb

```diff
@@ -48,10 +43,15 @@ def cac_by_address
       @cac_by_address ||=
         CAC.cac_data.each_with_object({}) do |entry_raw, acc|
           entry = NoWarningMash.new(entry_raw)
-          address = [entry.location_address_street,
-                     entry.location_address_locality,
-                     entry.location_address_region,
-                     entry.location_address_postal_code].join(SEPARATOR)
+          address = entry.location_address_street
+
+          # Seems that CAC is returning null for everything other than
+          # street. Hopefully they'll start providing the structured data
+          # again.
+          # address = [entry.location_address_street,
+          #            entry.location_address_locality,
+          #            entry.location_address_region,
+          #            entry.location_address_postal_code].join(SEPARATOR)
```

Comment:
Could we add `skip` to an appropriate test to cover this?

Reply:
Hmm not sure what we'd be testing. Feels like this would be a test of the CAC API, not our own code?